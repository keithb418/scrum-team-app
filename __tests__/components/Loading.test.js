import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../../front-end/src/components/Loading';

test('should render the Loading component correctly', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
});