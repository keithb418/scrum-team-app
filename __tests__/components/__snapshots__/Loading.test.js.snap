// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Loading component correctly 1`] = `
<div
  className="loading"
>
  <FontAwesome
    name="refresh"
    size="4x"
    spin={true}
  />
  <span
    className="sr-only"
  >
    Loading...
  </span>
</div>
`;
